--- routes/web.php
+++ routes/web.php
@@ -1683,6 +1683,7 @@
             Route::get('{contratos}/{state}/state_oltcatv_lote', 'ContratosController@state_oltcatv_lote')->name('contratos.state_oltcatv_lote');
             Route::get('{contratos}/enviar_mk_lote', 'ContratosController@enviar_mk_lote')->name('contratos.enviar_mk_lote');
             Route::get('{contratos}/{server_configuration_id}/{plan_id}/planes_lote', 'ContratosController@planes_lote')->name('contratos.planes_lote');
+            Route::get('{contratos}/destroy_lote', 'ContratosController@destroy_lote')->name('contratos.destroy_lote');
             Route::post('/enviar-eventos', 'ContratosController@enviarEventos');
             Route::post('/eventos-wifi', 'ContratosController@updateEventosWifi');
         });
